extensions/adobe/experience/campaign/notificationSubscriptionTarget.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":
+    "https://ns.adobe.com/experience/campaign/notificationSubscriptionTarget",
```

Comment:
I believe we are making schema URI all lower case

Reply:
Corrected the naming and reference